lab-matt/test/model-route-test.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
```

Comment:
Your tests are good, but could be significantly refactored.  Your tests should generate all the data they'll need at runtime and delete everything they create when they're finished.  That way each test run will get a fresh start.  I suggest in the future you do them in the following order:

GENERAL - 404
POST
GET
PUT
DELETE

You only need to create one note, in your initial POST request, using that you can get it and delete it to test the GET and DELETE routes 200 cases, the 400/404 tests won't need reference to the note anyway.  This way you cut down on the length/complexity of your tests and since you delete the note last you shouldn't run into false-positive pitfalls.